.clang-format

```diff
@@ -19,7 +19,7 @@
 BasedOnStyle: Google
 IndentWidth: 4
 ColumnLimit: 110
-SortIncludes: false
+SortIncludes: true
```

Comment:
The value should be `CaseInsensitive`  or `CaseSensitive `, see https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
Currently the `clang-format` binds to 11.0.0, this option is a boolean value, see https://releases.llvm.org/11.0.1/tools/clang/docs/ClangFormatStyleOptions.html

Reply:
Interesting. Looks like the `clang-format version 11.1.0` changed the option to boolean, and the `clang-format version 14.0.6` changed back. 

```
➜   clang-format -version
clang-format version 14.0.6
➜  clang-format -style=llvm -dump-config | grep SortIncludes
SortIncludes:    CaseSensitive

➜  clang-format-11 -version
clang-format version 11.1.0
➜  clang-format-11 -style=llvm -dump-config | grep SortIncludes
SortIncludes:    true
```

Reply:
No. This option is a boolean since the `clang-format` 3.9. It's a newer version of clang that changed it.

Reply:
https://releases.llvm.org/3.9.0/tools/clang/docs/ClangFormatStyleOptions.html